insecure/integration/test/composability_test.go

```diff
@@ -42,57 +41,56 @@ func TestCorruptibleConduitFrameworkHappyPath(t *testing.T) {
 		// extracting port that ccf gRPC server is running on
 		_, ccfPortStr, err := net.SplitHostPort(ccf.ServerAddress())
 		require.NoError(t, err)
-		ccfPort, err := strconv.Atoi(ccfPortStr)
-		require.NoError(t, err)
 
-		withAttackOrchestrator(t, flow.IdentityList{&corruptedIdentity}, ccfPort, func(event *insecure.Event) {
-			// implementing the corruption functionality of the orchestrator.
-			event.FlowProtocolEvent = corruptedEvent
-		}, func(t *testing.T) {
-			hub := stub.NewNetworkHub()
-			originalEvent := &message.TestMessage{Text: "this is a test message"}
-			testChannel := flownet.Channel("test-channel")
-
-			// corrupted node network
-			corruptedEngine := &network.Engine{}
-			corruptedNodeNetwork := stub.NewNetwork(t, corruptedIdentity.NodeID, hub, stub.WithConduitFactory(ccf))
-			corruptedConduit, err := corruptedNodeNetwork.Register(testChannel, corruptedEngine)
-			require.NoError(t, err)
-
-			// honest network
-			honestIdentity := unittest.IdentityFixture()
-			honestEngine := &network.Engine{}
-			honestNodeNetwork := stub.NewNetwork(t, honestIdentity.NodeID, hub)
-			// in this test, the honest node is only the receiver, hence, we discard
-			// the created conduit.
-			_, err = honestNodeNetwork.Register(testChannel, honestEngine)
-			require.NoError(t, err)
-
-			wg := &sync.WaitGroup{}
-			wg.Add(1)
-			honestEngine.OnProcess(func(channel flownet.Channel, originId flow.Identifier, event interface{}) error {
-				// implementing the process logic of the honest engine on reception of message from underlying network.
-				require.Equal(t, testChannel, channel)               // event must arrive at the channel set by orchestrator.
-				require.Equal(t, corruptedIdentity.NodeID, originId) // origin id of the message must be the corrupted node.
-				require.Equal(t, corruptedEvent, event)              // content of event must be swapped with corrupted event.
-
-				wg.Done()
-				return nil
+		withAttackOrchestrator(t, flow.IdentityList{&corruptedIdentity}, map[flow.Identifier]string{corruptedIdentity.NodeID: ccfPortStr},
```

Comment:
not a major change, just refactoring to have the port mapping for corrupted nodes.